radis/api/hitempapi.py

```diff
@@ -345,6 +427,18 @@ def parse_to_local_file(
 
         writer = self.get_datafile_manager()
 
+        if molecule == "CO2":
+            session = login_to_hitran()
+            if session:
+                download_hitemp_file(
+                    session,
+                    "https://hitran.org/files/HITEMP/bzip2format/02_HITEMP2024.par.bz2",
+                    "02_HITEMP2024.par.bz2",
+                )
+                urlname = "02_HITEMP2024.par.bz2"
+            else:
+                return 0  # Exit if login failed
```

Comment:
Why is an if statement required? You could directly do 
```suggestion
                download_hitemp_file(
                    session,
                    "https://hitran.org/files/HITEMP/bzip2format/02_HITEMP2024.par.bz2",
                    "02_HITEMP2024.par.bz2",
                )
                urlname = "02_HITEMP2024.par.bz2"
```